packages/integrations/tailwind/src/index.ts

```diff
@@ -47,10 +47,10 @@ async function getUserConfig(root: URL, configPath?: string, isRestart = false)
 				cwd: resolvedRoot,
 				filePath: tempConfigPath,
 			});
-
+		} catch (err) {
+			console.error(err)
```

Comment:
I found myself checking the terminal more often than the editor for errors, so would like to print the error here.